src/LanguageServer.Engine/Documents/Workspace.cs

```diff
@@ -172,13 +172,11 @@ public async Task<ProjectDocument> GetProjectDocument(Uri documentUri, bool relo
                 isNewProject = true;
 
                 if (MasterProject == null)
-                {
-                    MasterProject = new MasterProjectDocument(this, documentUri, Log);
-                    return MasterProject;
-                }
+                    return MasterProject = new MasterProjectDocument(this, documentUri, Log);
```

Comment:
Please don't use assignment expressions like this. If I remember correctly, we already had problems with this before when I accidentally removed such assignment without noticing

Reply:
Fair enough; I don’t feel particularly strongly about their usage but if they’re already causing problems I’m happy to remove it.